pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -0,0 +1,21 @@
+{ picom, lib, fetchFromGitHub, pcre }:
+picom.overrideAttrs (finalAttrs: previousAttrs: {
+  pname = "picom-pijulius";
+  version = "2024-05-17-unstable";
+
+  src = fetchFromGitHub {
+    owner = "pijulius";
+    repo = "picom";
+    rev = "e7b1488";
```

Please use the full rev hash in this case 
```suggestion
    rev = "e7b14886ae644aaa657383f7c4f44be7797fd5f6";
```